packages/web-components/src/components/va-table/va-table-slot.css

```diff
@@ -13,3 +13,9 @@ va-table > va-table-row[slot] button:hover {
 va-table > va-table-row[slot] button svg {
   width: 2.4rem;
 }
+
+@media screen and (min-width: 768px) {
+  va-table-row span.text-align-right {
+    text-align: right!important;
+  }
+}
```

Comment:
This is exactly the same styles as what the previous `medium-screen:vads-u-text-align--right` class was appyling.